packages/devtools_app/lib/src/initializer.dart

```diff
@@ -117,7 +117,7 @@ class _InitializerState extends State<Initializer>
 
   /// Shows a "disconnected" overlay if the [service.serviceManager] is not currently connected.
   void _handleNoConnection() {
-    WidgetsBinding.instance.addPostFrameCallback((_) {
+    WidgetsBinding.instance.scheduleFrameCallback((_) {
```

Comment:
I don't understand why we were using a postFrameCallback. A frame might not render and then it wouldn't be called till some strange later time.